src/cnguts.h

```diff
@@ -194,7 +194,7 @@ extern SI cnRunSerial;			// incremented in cgInit; copied to Topi.runSerial in c
 extern VROUTINFO* UnspoolInfo;	// dm array of info re unspooling virtual reports into actual report files.
 								//  set up in cncult.cpp; format is for vrUnspool. vrpak.h struct.
 
-//---------------------- CGSOLAR.CPP PUBLIC VARIABLE ------------------------
+//---------------------- cgsolar.cpp PUBLIC VARIABLE ------------------------
```

Comment:
This is a tricky one. The file name is now consistent, but it is not consistent with the rest of the context here.

Reply:
I'm not saying it "needs" to be one way or the other. I'm just pointing out the new inconsistency you introduced and trying to understand why you came to the conclusion you did. I'm not saying it's wrong. Ultimately this specific case doesn't matter much at all. What really matters is that we are conscious of inconsistencies (even the more minor ones) and address them if possible.

Reply:
I think this is a non-issue compared to the fact that we have comments that say _PUBLIC VARIABLES_ and _FUNCTION DECLARATIONS_ at all.